src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -695,8 +695,8 @@ bool KivaManager::setupKivaInstances()
 					exposedFraction = expPerimMap[surfNum].exposedFraction;
 				}
 			} else {
-				userSetExposedPerimeter = false;
-				useDetailedExposedPerimeter = true;
```

Comment:
Good catch. This works on Clang, but not MSVC. I'm surprised the compiler didn't cast a warning, though.